src/platform/platform.ts

```diff
@@ -574,40 +574,43 @@ export class Platform {
     if (this._isPortrait === null || this._isPortrait === false && this._win['innerWidth'] < this._win['innerHeight']) {
       var win = this._win;
 
+      const innerWidth = win['innerWidth'];
```

Comment:
can you make this `var`? the transpiler sometimes generate suboptimal code when there are nested const/let variables.